### PR TITLE
Roundstart tip changes.

### PR DESCRIPTION
**Pull request by Pratepresidenten**
Removes some unused and wrong tips.
Changes to incorrect tips

### CODE REVIEW

**Comment by Pratepresidenten** [comment]
!review

File: config/example/tips.txt [hunk 1]
```diff
@@ -139,7 +127,7 @@ If you see this message something has gone seriously wrong.
 If you're having trouble as an antagonist, consider a Dionaea whitelist.
 As a Physician, you can attach limbs from one species to the torso of another species. You too can make your own Frankenstein monster!
 The second antag contest is still running, you know.
-Killing mice on sight is not and never will be gank, no matter how much people complain.
+Killing rats on sight is not and never will be gank, no matter how much people complain.
```
Comment (ParadoxSpace) [inline config/example/tips.txt]: excuse me

Comment (Geevies) [reply config/example/tips.txt]: Valid.

File: config/example/tips.txt [hunk 2]
```diff
@@ -53,22 +51,19 @@ As the Warden, you can use handcuffs on orange prisoner shoes to turn them into
...
-As the Chaplain, your bible is also a container that can store small items. Depending on your god, your starting bible may come with a surprise!
 As the Chaplain, you are much more likely to get a response by praying to the gods than most people. To boost your chances, make altars with colorful crayon runes, lit candles, and wire art.
 As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic seeds. These seeds can alternatively be ordered from cargo.
 As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
-As a Cook, any food you make will be much healthier than the junk food found in vendors. Having the crew routinely eating from you will provide minor buffs.
```
Comment (Arrow768) [inline config/example/tips.txt]: That's not exactly wrong/incorrect.

File: config/example/tips.txt [hunk 3]
```diff
@@ -13,9 +13,8 @@ Holding alt and left clicking a tile will allow you to see its contents in the t
 The resist button will allow you to resist out of handcuffs, being buckled to a chair or bed, out of locked lockers and more. Whenever you're stuck, try resisting!
 You can move an item out of the way by dragging it and then clicking on an adjacent tile with an empty hand.
 Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using. There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
-As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your unique lasgun or your life are things to worry about.
+As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to mercenaries, to traitors that need to rob you of your unique lasgun or your life are things to worry about.
```
Comment (Arrow768) [inline config/example/tips.txt]: lasgun

File: config/example/tips.txt [hunk 4]
```diff
@@ -53,22 +51,19 @@ As the Warden, you can use handcuffs on orange prisoner shoes to turn them into
...
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
-As a Cook, any food you make will be much healthier than the junk food found in vendors. Having the crew routinely eating from you will provide minor buffs.
 As the Bartender, the drinks you start with only give you the basics. If you want more advanced mixtures, look into working with chemistry, hydroponics, or even mining for things to grind up and throw in!
 As the Bartender, you can use a circular saw on your shotgun to make it easier to store.
 As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer or order another from Cargo.
 As a Janitor, mousetraps can be used to create bombs or booby-trap containers.
 As the Librarian, be sure to keep the shelves stocked and the library clean for crew.
-As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
```
Comment (Arrow768) [inline config/example/tips.txt]: Not incorrect

File: config/example/tips.txt (hunk 2 above)
Comment (Pratepresidenten) [reply config/example/tips.txt]: How so? There is no mechanical benefit from eating food prepped at the kitchen?

File: config/example/tips.txt (hunk 4 above)
Comment (Pratepresidenten) [reply config/example/tips.txt]: Kinda is though, as mulebots does not exist right now. They've been gone since.. Early releases of Aurora?

**Pull request merged**